sql-cli/noxfile.py

```diff
@@ -21,15 +21,45 @@ def dev(session: nox.Session) -> None:
 
 
 @nox.session(python=["3.7", "3.8", "3.9"])
-def test(session: nox.Session) -> None:
+@nox.parametrize("airflow", ["2.2.5", "2.3.4", "2.4.2"])
+def test(session: nox.Session, airflow: str) -> None:
     """Run both unit and integration tests."""
+
     session.install("poetry")
-    session.run("poetry", "install", "--with", "dev")
-    # Log all the installed dependencies
+    if airflow.startswith("2.2"):
```

Comment:
I'd rather we do this for all 2.2 - other versions also need to be constraint, and in case we need to test any other specific 2.2, we would not need to change the comparison.